lib/salus/scanners/report_go_dep.rb

```diff
@@ -36,7 +36,7 @@ def record_dep_from_go_sum
             {
               "namespace" => (matches[:namespace]).to_s,
               "name" => (matches[:name]).to_s,
-              "version" => (matches[:version]).to_s.gsub(%r{/go.mod}, '').strip,
+              "version" => (matches[:version]).to_s.gsub(/^v/, '').gsub(%r{/go.mod}, '').strip,
```

Comment:
I assume dropping v from the start of the versions won't impact anything with dependency track?

Reply:
Would be nice to double check with the search option here https://ossindex.sonatype.org/

Reply:
Will revert just to be completely safe 👍